packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -110,6 +111,13 @@ export class CXLMarketingNavElement extends LitElement {
     `;
   }
 
+  /**
+   * Hide search when minimal view is enabled
+   */
+  isSearchHidden() {
```

Comment:
Avoid creating "used once" new symbols.

Can we write what this method does directly into template `?hidden` condition?